cdap-ui/app/services/constants.js

```diff
@@ -31,5 +31,32 @@ angular.module(PKG.name + '.services')
         'transform': 'transform'
       }
     },
-    addHydratorAppLabel: 'Hydrator Pipeline'
+
+    'en': {
+      hydrator: {
+        appLabel: 'Hydrator Pipeline',
+        studio: {
+          oneSinkError: 'Please add at least one sink to your pipeline.',
+          sourceRequiredFieldsError: 'Please provide required fields for Source.',
+          sinkRequiredFieldsError: 'Please provide required fields for Sink.',
+          transformRequiredFieldsError: 'Please provide required fields for Transform.',
+          oneSourceError: 'Pipelines can only have one source. Please remove any additional sources.',
+          noSourceError: 'Please add a source to your pipeline',
+          nameError: 'Please name your pipeline.',
+          nameValidationError: 'Pipeline names can only contain letters of the alphabet, numbers, and \'_\' Please remove other character types.',
```

Comment:
Pipeline names can only contain alphanumeric (\'a-z A-Z 0-9\') characters and underscores ( \'_\'). Please remove other characters.